src/modules/85-cv/pages/monitored-service/components/Configurations/Configurations.tsx

```diff
@@ -174,6 +174,9 @@ export default function Configurations(
         cloneTemplateValue.spec.sources['changeSources'] = defaultMonitoredService?.sources?.changeSources
         sethasTemplateChangeSourceSet(true)
       }
+      if (defaultMonitoredService?.type) {
+        cloneTemplateValue.spec['type'] = defaultMonitoredService?.type
```

Comment:
NIT: This conditional chaining can be removed as we are already checking for the value. 
```suggestion
        cloneTemplateValue.spec['type'] = defaultMonitoredService.type
```